doc/content/lorawan/antenna-types/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Antenna Types"
+section: Additional Information
+description: ""
+weight:
+---
+
+In this tutorial, you will learn about some of the popular antenna types used with LoRaWAN end devices and gateways.
+
+## Wire Antenna
+
+Wire antennas are low-cost and easy to build and work well in indoor environments. They are usually quarter-wave antennas. Different frequencies need different wire lengths (calculated for quarter-wave), for example, 3.25 inches or 8.2 cm for 868 MHz and 3 inches or 7.8 cm for 915 MHz. Suitable for DIY projects. You must keep the antenna wire as vertical as possible for better performance.
+
+{{< figure src="wire-antenna-example.jpg" class="plain" >}}
+image credits: [SparkFun Electronics](https://learn.sparkfun.com/)
```

Comment:
If you use _image courtesy of ..._ in one page, please be consistent and use the same in others as well